### PR TITLE
win_bash_run

### PR DESCRIPTION
**Pull request by memsharded**

### CODE REVIEW

File: reference/conanfile/attributes.rst [hunk 1]
```diff
@@ -1809,6 +1809,32 @@ When ``True`` it enables the new run in a subsystem bash in Windows mechanism. :
...
+win_bash_run
+------------
+
+.. warning::
+
+    This is an **experimental** feature subject to breaking changes in future releases.
+
+Available since: `1.54.0 <https://github.com/conan-io/conan/releases/tag/1.54.0>`_
+
+When ``True`` it enables running commands in the ``"run"`` scope, to run them inside a bash shell.
```
Comment (SSE4) [inline reference/conanfile/attributes.rst:1825]: how do I enable it for an another scope?

File: migrating_to_2.0/recipes.rst [hunk 2]
```diff
@@ -949,8 +949,12 @@ You need to configure how to run the commands with two config variables:
 
     - **tools.microsoft.bash:subsystem**:  Possible values: 'msys2', 'msys', 'cygwin', 'wsl' and 'sfu'
     - **tools.microsoft.bash:path** (Default "bash"): Path to the shell executable.
+    - **tools.microsoft.bash:active** (Default "None"): Used to define if Conan is already running inside a subsystem (Msys2) terminal.
 
 Any command run with ``self.run``, if ``self.win_bash == True`` will run the command inside the specified shell.
+Any command run with ``self.run(..., scope="run")`` if ``self.win_bash_run == True`` will run that command inside the shell.
+In both cases running explicitly in the bash shell only happens if ``tools.microsoft.bash:active`` is not True, because
+when it is True, it means that COnan is already running inside the shell.
```
Comment (SSE4) [inline migrating_to_2.0/recipes.rst:957]:
```suggestion
when it is True, it means that Conan is already running inside the shell.
```

File: reference/conanfile/attributes.rst (hunk 1 above)
Comment (memsharded) [reply reference/conanfile/attributes.rst:1825]: ``win_bash`` is that is used for the other scope ("build")

File: migrating_to_2.0/recipes.rst [hunk 3]
```diff
@@ -949,8 +949,12 @@ You need to configure how to run the commands with two config variables:
```
Comment (czoido) [inline migrating_to_2.0/recipes.rst:949]: should be _with three_ now

File: migrating_to_2.0/recipes.rst [hunk 4]
```diff
@@ -949,8 +949,12 @@ You need to configure how to run the commands with two config variables:
 
     - **tools.microsoft.bash:subsystem**:  Possible values: 'msys2', 'msys', 'cygwin', 'wsl' and 'sfu'
     - **tools.microsoft.bash:path** (Default "bash"): Path to the shell executable.
+    - **tools.microsoft.bash:active** (Default "None"): Used to define if Conan is already running inside a subsystem (Msys2) terminal.
 
 Any command run with ``self.run``, if ``self.win_bash == True`` will run the command inside the specified shell.
+Any command run with ``self.run(..., scope="run")`` if ``self.win_bash_run == True`` will run that command inside the shell.
+In both cases running explicitly in the bash shell only happens if ``tools.microsoft.bash:active`` is not True, because
+when it is True, it means that Conan is already running inside the shell.
```
Comment (czoido) [inline migrating_to_2.0/recipes.rst:957]: For me this part reads a bit confusing, maybe illustrating with what Conan calls for each case or some example would make this part a bit more clear?

File: reference/conanfile/attributes.rst (hunk 1 above)
Comment (czoido) [reply reference/conanfile/attributes.rst:1825]: I think that's not completely clear, maybe we can add some clarification on that for `win_bash` and maybe adding here something like this is the equivalent of `win_bash` for the `run` scope ?

**Pull request merged**